src/zmqpp/socket.cpp

```diff
@@ -341,7 +341,7 @@ bool socket::receive_raw(char* buffer, size_t& length, int const flags /* = NORM
 
 	if(result >= 0)
 	{
-		length = std::min(length, zmq_msg_size(&_recv_buffer));
+		length = (std::min<size_t>)(length, zmq_msg_size(&_recv_buffer));
```

Comment:
Why is this wrapped in parenthesis, shouldn't it just be `std::min<size_t>(length, zmq_msg_size(&_recv_buffer))`?

Reply:
to avoid min macro expansion (from windows.h)